test/end-to-end/rubrics/rubrics.spec.js

```diff
@@ -0,0 +1,48 @@
+const helpers = require('../shared/helpers');
+const RubricPage = require('./rubrics.page');
+const chai = require('chai');
+
+
+/** configuring helpers**/
+helpers.configure(chai);
+
+describe('Rubrics Management', () => {
+  // navigate to the page
+  before(() => helpers.navigate('#!/payroll/rubrics'));
+
+  const Page = new RubricPage();
+
+  const rubric = {
+    label : 'Rubric Syndical',
+    abbr  : 'CoSynd',
+    is_employee : 1,
+    is_percent : 1,
+    third_party_account_id : '40111002',
```

Comment:
Could you include a comment with the account name here?  Like:
```
 third_party_account_id : '40111002', // SNEL
```
or something?  That will help future developers know what account's balance is changing.